scripts/update_schemastore.py

```diff
@@ -138,7 +138,8 @@ def determine_git_protocol(argv: list[str] | None = None) -> GitProtocol:
         "--proto",
         required=True,
```

Comment:
Should we still mark it as required if it now has a default? I forget how this works with argparse and always have to look it up

Reply:
Oops